tests/test_http_graphql_schema.py

```diff
@@ -2495,6 +2505,404 @@ def test_graphql_schema_type_19(self):
             }
         })
 
+    def test_graphql_schema_type_20(self):
+        # make sure the union type got reflected
+        self.assert_graphql_query_result(r"""
+        query {
+            __type(name: "Profile_or_Setting") {
```

Comment:
Do we really want to support this? Supporting it as a link target is important but supporting it directly isn't really.
Or does graphql basically require we support it if we support it as a link target?

Reply:
This is the type that is returned by the link.

Reply:
Right, but does that mean we need to actually expose it as a standalone thing that you can select directly?

Reply:
This test checks the available types, which should exist. The `INTERFACE` is basically an abstract return type.

There's another test a bit later `test_graphql_reflection_03` that actually checks that the corresponding queries or mutations don't get implemented, which is how we know it's not available for direct querying.